frameworks/hdfs/alerts/HDFS_alert_rules.yml

```diff
@@ -0,0 +1,41 @@
+groups:
+- name: critical
+  rules:
+
+  - alert: InstanceDown
+    expr: up == 0
+    for: 5m
+    labels:
+      severity: page
+    annotations:
+      summary: "Instance {{ $labels.instance }} down"
+      description: "{{ $labels.instance }} of job {{ $labels.job }} has been down for more than 5 minutes."
+
+  # Alert for any instance that has a median request latency > 1s.
+  - alert: APIHighRequestLatency
+    expr: api_http_request_latencies_second{quantile="0.5"} > 1
+    for: 1s
+    annotations:
+      summary: "High request latency on {{ $labels.instance }}"
+      description: "{{ $labels.instance }} has a median request latency above 1s (current value: {{ $value }}s)"
+
+  - alert: NumInactiveSources
+    expr: (sum({__name__=~".*metricssystem_MetricsSystem_NumAllSources"}) - sum({__name__=~".*metricssystem_MetricsSystem_NumActiveSources"})) > 0
+    for: 10m
+    annotations:
+      summary: "Number of inactive sources for {{ $labels.instance }}"
+      description: "{{ $labels.instance }} has number of inactive sources greater then 0 (current value: {{ $value }})"
+
+  - alert: MemHeapUsed
+    expr: (sum({__name__=~".*_jvm_JvmMetrics_MemHeapMaxM"}) / sum({__name__=~".*_jvm_JvmMetrics_MemHeapUsedM"})) < 0.8
+    for: 10m
+    annotations:
+      summary: "Total service memory heap in used for {{ $labels.instance }}"
```

Comment:
small typo: should be `Total service memory heap in use ...`. Same thing for L3, 40, and 41

Reply:
yeah, thanks